platform/linux-generic/arch/aarch64/odp/api/abi/hash.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2021 ARM Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_API_ABI_HASH_H_
+#define ODP_API_ABI_HASH_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef __ARM_FEATURE_CRC32
+#define _ODP_CRC32_EXTENSIONS
+
+#include <arm_acle.h>
+
+static inline uint32_t _crc32_hash(const void *data_ptr, uint32_t data_len,
+				   uint32_t init_val)
```

Comment:
Okay, I think then it will become consistent with the implementation in platform/linux-generic/odp_hash_crc32c.c. I will make the change. Thanks Jere!